packages/botonic-plugin-contentful/package.json

```diff
@@ -33,18 +33,20 @@
   "license": "MIT",
   "dependencies": {
     "@babel/runtime": "^7.11.2",
-    "@nlpjs/core": "^4.11.0",
-    "@nlpjs/lang-ca": "^4.11.0",
-    "@nlpjs/lang-en-min": "^4.11.0",
-    "@nlpjs/lang-es": "^4.11.0",
-    "@nlpjs/lang-fr": "^4.11.0",
-    "@nlpjs/lang-it": "^4.11.0",
-    "@nlpjs/lang-pl": "^4.11.0",
-    "@nlpjs/lang-pt": "^4.11.0",
-    "@nlpjs/lang-ru": "^4.11.0",
-    "@nlpjs/lang-tr": "^4.11.0",
-    "@nlpjs/ner": "^4.11.0",
-    "@nlpjs/similarity": "^4.11.0",
+    "@nlpjs/core": "^4.14.0",
+    "@nlpjs/lang-ca": "^4.14.0",
+    "@nlpjs/lang-en-min": "^4.14.0",
+    "@nlpjs/lang-es": "^4.14.0",
+    "@nlpjs/lang-de": "^4.14.0",
+    "@nlpjs/lang-fr": "^4.14.0",
+    "@nlpjs/lang-it": "^4.14.0",
+    "@nlpjs/lang-pl": "^4.14.0",
+    "@nlpjs/lang-pt": "^4.14.0",
+    "@nlpjs/lang-ro": "^4.14.0",
+    "@nlpjs/lang-ru": "^4.14.0",
+    "@nlpjs/lang-tr": "^4.14.0",
+    "@nlpjs/ner": "^4.14.0",
+    "@nlpjs/similarity": "^4.14.0",
```

Comment:
I see no usages of similarity in the code, was this missing for any other module?

Reply:
it's there. you probably did your "find" with wrong configuration